rimage/transform/brown_conrady_test.go

```diff
@@ -0,0 +1,16 @@
+package transform
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+)
+
+func TestBrownConradyCheckValid(t *testing.T) {
+	distortionsA := &BrownConrady{}
+	test.That(t, distortionsA.CheckValid(), test.ShouldBeNil)
+	var nilBrownConradyPtr *BrownConrady
+	err := nilBrownConradyPtr.CheckValid().Error()
+	expected := "BrownConrady shaped distortion_parameters not provided: invalid distortion_parameters"
+	test.That(t, err, test.ShouldContainSubstring, expected)
```

Comment:
Could you move the .Error() to be on the `err` in line 15? If for some reason, the test fails and CheckValid() returns nil, then line 13 will panic.

Reply:
Note: `err.Error()`